Platforms/QemuSbsaPkg/MsSecurePartition/MsSecurePartition.c

```diff
@@ -0,0 +1,115 @@
+/** @file
+  Microsoft Secure Partition
+
+  Copyright (c), Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+// Secure Partition Headers
+#include <PiMm.h>
+#include <Base.h>
+#include <IndustryStandard/ArmFfaSvc.h>
+
+#include <Library/StandaloneMmCoreEntryPoint.h>
+#include <Library/DebugLib.h>
+#include <Library/BaseMemoryLib.h>
+#include <Library/IoLib.h>
+#include <Library/PcdLib.h>
+#include <Library/ArmSvcLib.h>
+#include <Library/ArmFfaLib.h>
+#include <Library/ArmFfaLibEx.h>
+#include <Library/NotificationServiceLib.h>
+#include <Library/TestServiceLib.h>
+#include <Library/TpmServiceLib.h>
+#include <Guid/Tpm2ServiceFfa.h>
+
+// Service specific structures/variables
+EFI_GUID NotificationServiceGuid = NOTIFICATION_SERVICE_UUID;
+EFI_GUID TestServiceGuid = TEST_SERVICE_UUID;
+volatile BOOLEAN Loop = TRUE;

```

Comment:
left over? :)

Reply:
No, I liked having this rather than a while (1) for debugging purposes. :) I would like to keep it.

Reply:
I think it is probably better to use something like `ASSERT(FALSE);` instead of `while (loop)`.